server/datastore/mysql/software_titles.go

```diff
@@ -219,11 +219,9 @@ GROUP BY st.id, software_package`
 		cveJoinType = "INNER"
 	}
 
-	var globalOrTeamID uint
-	args := []any{0}
+	args := []any{0, 0}
 	if opt.TeamID != nil {
-		args[0] = *opt.TeamID
-		globalOrTeamID = *opt.TeamID
+		args[0], args[1] = *opt.TeamID, *opt.TeamID
```

Comment:
this works withe current business logic because we know there can't be any rows with `global_or_team_id = 0`, right?

Reply:
For `software_installers`, that's correct (there will be a row with `0` for `software_titles_host_counts` and that's ok). It would also work if we decided to support "no team" installers eventually (and used `0` as `gloal_or_team_id` for that, as we do elsewhere).